cmp/grid/Grid.js

```diff
@@ -409,12 +409,18 @@ class LocalModel extends HoistModel {
                     this.updatePinnedSummaryRowData();
 
                     if (transaction?.update) {
+                        // Refresh cells in columns with complex renderers
                         const refreshCols = model.columns.filter(c => !c.hidden && c.rendererIsComplex);
                         if (refreshCols) {
                             const rowNodes = compact(transaction.update.map(r => agApi.getRowNode(r.id))),
                                 columns = refreshCols.map(c => c.colId);
                             agApi.refreshCells({rowNodes, columns, force: true});
                         }
+
+                        // Refresh row heights if autoHeight is enabled
+                        if (agApi.gridOptionsWrapper.columnController.isAutoRowHeightActive()) {
```

Comment:
This looks like it might be a private (or at least undocumented) method - at least I couldn't find a reference to it in the online ag-grid docs. I'm not opposed to us calling it in this lower-level Hoist code, but it does raise the possibility of it changing in the future in some tricky way.

What about checking if our own `autoHeight` property is true on any (visible) column?

Reply:
Thanks for taking a look! Good idea about checking the column's own autoHeight prop - seems safer all round. Will update

Reply:
Great, thanks!